packages/nodes-base/nodes/Gotify/Gotify.node.ts

```diff
@@ -115,6 +115,38 @@ export class Gotify implements INodeType {
 					},
 				],
 			},
+			{
+				displayName: 'Options',
+				name: 'options',
+				type: 'collection',
+				placeholder: 'Add Option',
+				displayOptions: {
+					show: {
+						resource: ['message'],
+						operation: ['create'],
+					},
+				},
+				default: {},
+				options: [
+					{
+						displayName: 'Content Type',
+						name: 'contentType',
+						type: 'options',
+						default: 'text/plain',
+						description: 'The message content type',
+						options: [
+							{
+								name: 'Plain',
+								value: 'text/plain',
+							},
+							{
+								name: 'Markdown',
+								value: 'text/markdown',
+							},
+						],
+					},
+				],
+			},
```

Comment:
@Joffcom 
should we have this selector on top level, like in email nodes?

Reply:
@michael-radency I did think about that but I think it is better as an option as it is not really required in the same way that we have it hidden for Telegram as well.